media/js/privacy/firefox-privacy.js

```diff
@@ -280,9 +284,11 @@
         }
     });
 
-    /* add a class to indicate that js is enabled. This will trigger
-    the appropriate styling to be applied */
-    mainContent.classList.add('interactive-widget');
-
-    initPage();
+    // do not execute in IE8 and lower
+    if (document.addEventListener) {
```

Comment:
Nit: change this to `'addEventListener' in window` (is this single detect enough?)

Also, I believe even with this check IE will throw an error since you have `mainContent.addEventListener('click')` above.

Reply:
Will do.

> is this single detect enough?
Yes, this will filter out IE8 and below

> Also, I believe even with this check IE will throw an error since you have mainContent.addEventListener('click') above.

True, I can move that into the `if` conditional to avoid that. Good call.